static/bidder-info/ucfunnel.yaml

```diff
@@ -13,5 +13,6 @@ capabilities:
       - video
 userSync:
   redirect:
+  # for syncing to work, please register your Prebid Server domain with the contact email above
```

Comment:
@bretg  should this also be mentioned in their bidder docs?

Reply:
@onkarvhanumante - the bidder docs are generally for publishers. This note is for Prebid Server host companies.